stories/scss/action-icon/action-icon-basic.stories.ts

```diff
@@ -0,0 +1,60 @@
+import { Meta, Story } from '@storybook/angular';
+
+interface ActionBasicStory {
+	mod: string;
+	palette: string;
+	state: string;
```

Comment:
A boolean here since you only have one possible state?

Reply:
✅